app/components/wallet/transactions/Transaction.js

```diff
@@ -224,6 +252,10 @@ export default class Transaction extends Component<Props, State> {
               </div>
             )}
             <div>
+              <h2>
+                Fee:
```

Comment:
it’s a really small detail but I would hide the `:` to be consistent with the other ones